src/jvm/test/resources/expected_logs/run_concurrent_test/deque_jdk8.txt

```diff
@@ -12,6 +12,7 @@ The following interleaving leads to the error:
 | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | $this_runCatching.block(): threw IllegalStateException at BaseRunConcurrentRepresentationTest.testRunWithModelChecker$lambda$1$lambda$0(RunConcurrentRepresentationTests.kt:47) |                                                                                                                                                  |                                                                                                                                                 |
 |   block(): threw IllegalStateException at IncorrectConcurrentLinkedDequeRunConcurrentRepresentationTest.block(RunConcurrentRepresentationTests.kt:349)                          |                                                                                                                                                  |                                                                                                                                                 |
+|     Node#1.item.putObject(null) at ConcurrentLinkedDeque$Node.<init>(ConcurrentLinkedDeque.java:303)                                                                            |                                                                                                                                                  |                                                                                                                                                 |
```

Comment:
Can we show constructors as method calls?

Reply:
What are the possible ways?

Reply:
I'm not sure but it should be very complicated as the constructor call is separated into two parts.

Reply:
@zhelenskiy, we already have such a logic for reads, wrapping them with "beforeRead" and "afterRead" methods. Would the same approach work here?

Reply:
no

Reply:
I have previously tried doing this for some time and failed, so I think that:
* It is not that simple.
* It is another [issue](https://youtrack.jetbrains.com/issue/IJTD-160/Support-constructors-in-MethodCallTransformer), and I don't see any reason to mix the PRs.